package.json

```diff
@@ -19,6 +19,12 @@
   "bugs": {
     "url": "https://github.com/katowulf/mockfirebase/issues"
   },
+  "files": [
+    "*.md",
+    "src/",
+    "tutorials/",
+    "browser/"
```

Comment:
Ahh, I see [that](https://docs.npmjs.com/files/package.json#files) now. Good to know. I guess the `README.md` would be picked up by `*.md` anyway. Okay cool.

My only last comment would be that I don't really think we need to deploy `/tutorials` in the npm package. I can't imagine anyone reading them from `/node_modules` after doing an npm install of this lib.